lib/client.js

```diff
@@ -859,6 +863,21 @@ class Parser {
       socket[kReset] = true
     }
 
+    for (let i = 0; i < headers.length; i+= 2) {
+      if (headers[i].toString().toLowerCase() === 'keep-alive') {
```

Comment:
I wonder what's the performance impact here...

Reply:
Does the parser already makes the headers lowercase?

Reply:
look how we do it in other places, compare with length first

Reply:
I finally did this like the `timeout` directive is done. PTAL :)